SECURITY.md

```diff
@@ -0,0 +1,75 @@
+ # Security Policy
+
+## Introduction
+
+Security researchers are essential in identifying vulnerabilities that may impact the Axelar Network. If you have discovered a security vulnerability in the Axelar chain or any repository managed by Axelar, we encourage you to notify us using one of the methods outlined below.
+
+### Guidelines for Responsible Vulnerability Testing and Reporting
+
+1. **Refrain from testing vulnerabilities on our publicly accessible environments**, including but not limited to:
+  - Axelar mainnet
+  - Axelar Frontend Apps e.g satellite.money
+  - Axelar Testnet
+  - Axelar Testnet Frontend Apps e.g testnet.satellite.money
+
+2. **Avoid reporting security vulnerabilities through public channels, including GitHub issues**
+
+## Reporting Security Issues
+
+To privately report a security vulnerability, please choose one of the following options:
+
+### 1. Email
+
+Send your detailed vulnerability report to `security@axelar.network`.
```

Comment:
should this be switched to interoplabs domain? if not, let's forward emails to interoplabs for convenience.